src/plugins/sessionPlayer/plugin.js

```diff
@@ -276,251 +274,227 @@ define(['playbackManager', 'events', 'serverNotifications', 'connectionManager']
+    subtitleTracks() {
+        let state = this.lastPlayerData || {};
         state = state.NowPlayingItem || {};
-        var streams = state.MediaStreams || [];
+        const streams = state.MediaStreams || [];
         return streams.filter(function (s) {
             return s.Type === 'Subtitle';
         });
-    };
+    }
 
-    SessionPlayer.prototype.getSubtitleStreamIndex = function () {
-        var state = this.lastPlayerData || {};
+    getSubtitleStreamIndex() {
+        let state = this.lastPlayerData || {};
         state = state.PlayState || {};
         return state.SubtitleStreamIndex;
-    };
+    }
 
-    SessionPlayer.prototype.setSubtitleStreamIndex = function (index) {
+    setSubtitleStreamIndex(index) {
         sendCommandByName(this, 'SetSubtitleStreamIndex', {
             Index: index
         });
-    };
-
-    SessionPlayer.prototype.getMaxStreamingBitrate = function () {
-
-    };
-
-    SessionPlayer.prototype.setMaxStreamingBitrate = function (options) {
-
-    };
-
-    SessionPlayer.prototype.isFullscreen = function () {
-
-    };
-
-    SessionPlayer.prototype.toggleFullscreen = function () {
-
-    };
-
-    SessionPlayer.prototype.getRepeatMode = function () {
-
-    };
+    }
 
-    SessionPlayer.prototype.setRepeatMode = function (mode) {
+    setRepeatMode(mode) {
         sendCommandByName(this, 'SetRepeatMode', {
             RepeatMode: mode
         });
-    };
+    }
 
-    SessionPlayer.prototype.setQueueShuffleMode = function (mode) {
+    setQueueShuffleMode(mode) {
         sendCommandByName(this, 'SetShuffleQueue', {
             ShuffleMode: mode
         });
-    };
-
-    SessionPlayer.prototype.getQueueShuffleMode = function () {
+    }
 
-    };
+    getQueueShuffleMode() {
+    }
 
-    SessionPlayer.prototype.displayContent = function (options) {
+    displayContent(options) {
         sendCommandByName(this, 'DisplayContent', options);
-    };
+    }
 
-    SessionPlayer.prototype.isPlaying = function () {
-        var state = this.lastPlayerData || {};
+    isPlaying() {
+        const state = this.lastPlayerData || {};
         return state.NowPlayingItem != null;
-    };
+    }
 
-    SessionPlayer.prototype.isPlayingVideo = function () {
-        var state = this.lastPlayerData || {};
+    isPlayingVideo() {
+        let state = this.lastPlayerData || {};
         state = state.NowPlayingItem || {};
         return state.MediaType === 'Video';
-    };
+    }
 
-    SessionPlayer.prototype.isPlayingAudio = function () {
-        var state = this.lastPlayerData || {};
+    isPlayingAudio() {
+        let state = this.lastPlayerData || {};
         state = state.NowPlayingItem || {};
         return state.MediaType === 'Audio';
-    };
+    }
 
-    SessionPlayer.prototype.getPlaylist = function () {
+    getPlaylist() {
         return Promise.resolve([]);
-    };
-
-    SessionPlayer.prototype.getCurrentPlaylistItemId = function () {
-    };
```

Comment:
It seems this one is required too.
https://github.com/jellyfin/jellyfin-web/blob/1339bb97f4161cddf2c9057a613c99f278e07bfc/src/components/playback/playbackmanager.js#L2562-L2564

Reply:
Added Back